ansible/roles/flux/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+
+---
+- name: "Do action '{{ do }}'' role"
+  include_tasks: "{{ do | lower }}.yml"
+...
```

Comment:
Does the '...' signify anything in Ansible file syntax?

Reply:
Not if it is in "....". E.g. "Do action '{{ do }}'' role" will output -> `TASK [flux : Do action 'create'' role]` . Just the additional `'` after the variable need to be removed.